### ISSUE TITLE
Error during build

### ISSUE THREAD

**Issue by holasidx** [mhekkel/mrc#16]
When I use the "cmake --build build" command, I get these error messages:

Error opening template file "/home/my_user/mrc/build/mrc-bootstrap and"
gmake[2]: *** [CMakeFiles/mrc.dir/build.make:75: mrc-rsrc.obj] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:894: CMakeFiles/mrc.dir/all] Error 2
gmake: *** [Makefile:146: all] Error 2

**Comment by HeJunhong1107** [comment]
I met the same error

**Comment by mhekkel** [comment]
What hardware/OS are you using?

**Comment by mhekkel** [comment]
> Error opening template file "/home/my_user/mrc/build/mrc-bootstrap and"

The word _and_ here is inside the double quotes, was that a typing error where you meant this word to indicate there are more errors or did it really end up at that spot?

**Comment by Sigmanificient** [comment]
Same issue here. Using nixos standard build env and it fails to build for version `1.3.11` & later

cmake logs:
```
-- Build files have been written to: /build/source/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
build flags: -j16 SHELL=/nix/store/m101dg80ngyjdb02g6jwy80sr7kzj26g-bash-5.2p26/bin/bash
[ 33%] Building CXX object CMakeFiles/mrc-bootstrap.dir/src/mrc.cpp.o
[ 33%] Building CXX object CMakeFiles/mrc-bootstrap.dir/src/dummy.cpp.o
[ 50%] Linking CXX executable mrc-bootstrap
[ 50%] Built target mrc-bootstrap
[ 66%] Generating mrc-rsrc.obj
Error opening template file "/build/source/build/mrc-bootstrap<"
make[2]: *** [CMakeFiles/mrc.dir/build.make:75: mrc-rsrc.obj] Error 1
make[1]: *** [CMakeFiles/Makefile2:84: CMakeFiles/mrc.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
error: builder for '/nix/store/f02wqdvgi4zfwpbzx7hhr78n255sq0jd-mrc-1.3.11.drv' failed with exit code 2;
```

cmake flags used (auto-generated from cmake hook):
```
-DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF
-DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF
-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON
-DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF
-DCMAKE_INSTALL_LOCALEDIR=/nix/store/a748cz5hd467rsq78d513zfggbgifwsi-alphafill-2.1.0/share/locale
-DCMAKE_INSTALL_LIBEXECDIR=/nix/store/a748cz5hd467rsq78d513zfggbgifwsi-alphafill-2.1.0/libexec
-DCMAKE_INSTALL_LIBDIR=/nix/store/a748cz5hd467rsq78d513zfggbgifwsi-alphafill-2.1.0/lib
-DCMAKE_INSTALL_DOCDIR=/nix/store/a748cz5hd467rsq78d513zfggbgifwsi-alphafill-2.1.0/share/doc/alphafill
-DCMAKE_INSTALL_INFODIR=/nix/store/a748cz5hd467rsq78d513zfggbgifwsi-alphafill-2.1.0/share/info
-DCMAKE_INSTALL_MANDIR=/nix/store/a748cz5hd467rsq78d513zfggbgifwsi-alphafill-2.1.0/share/man
-DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/a748cz5hd467rsq78d513zfggbgifwsi-alphafill-2.1.0/include
-DCMAKE_INSTALL_INCLUDEDIR=/nix/store/a748cz5hd467rsq78d513zfggbgifwsi-alphafill-2.1.0/include
-DCMAKE_INSTALL_SBINDIR=/nix/store/a748cz5hd467rsq78d513zfggbgifwsi-alphafill-2.1.0/sbin
-DCMAKE_INSTALL_BINDIR=/nix/store/a748cz5hd467rsq78d513zfggbgifwsi-alphafill-2.1.0/bin
-DCMAKE_INSTALL_NAME_DIR=/nix/store/a748cz5hd467rsq78d513zfggbgifwsi-alphafill-2.1.0/lib
-DCMAKE_POLICY_DEFAULT_CMP0025=NEW
-DCMAKE_OSX_SYSROOT=
-DCMAKE_FIND_FRAMEWORK=LAST
-DCMAKE_STRIP=/nix/store/kp2j7yn0wzwq5piy494r54dafrh83s6s-gcc-wrapper-13.3.0/bin/strip
-DCMAKE_RANLIB=/nix/store/kp2j7yn0wzwq5piy494r54dafrh83s6s-gcc-wrapper-13.3.0/bin/ranlib
-DCMAKE_AR=/nix/store/kp2j7yn0wzwq5piy494r54dafrh83s6s-gcc-wrapper-13.3.0/bin/ar
-DCMAKE_C_COMPILER=gcc
-DCMAKE_CXX_COMPILER=g++
-DCMAKE_INSTALL_PREFIX=/nix/store/a748cz5hd467rsq78d513zfggbgifwsi-alphafill-2.1.0
```

**Comment by mhekkel** [comment]
I should have read the first report here more closely. Thanks for sending the second one.

problem was that readlink is not writing a null character. Duh.

Fixed in 1.3.13
